### PR TITLE
update custom image upload docs

### PR DESCRIPTION
**Pull request by TylerGillson**
The `content` arg needs `@=`, rather than `=`, and `sha256` and `size` are required args using the MAAS CLI from the v3.0 snap.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/cluster-api-provider-maas/pull/44#" title="Author self-approved">TylerGillson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jzhoucliqr**
You can assign the PR to them by writing `/assign @jzhoucliqr` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Fcluster-api-provider-maas).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/cluster-api-provider-maas/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jzhoucliqr — APPROVED** [review]

**Comment by spectro-prow** [comment]
@TylerGillson: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

<details>

In response to [this](https://github.com/spectrocloud/cluster-api-provider-maas/pull/44#issuecomment-1317406387):

>/test all


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
